ehrenamt-justiz-eai/README.md

```
@@ -76,3 +76,26 @@ Change to the folder "ehrenamt-justiz-eai" and call the following scripts depend
+    format:
+      console: # set to empty to disable structured logging locally
+  level:
+    root: INFO
+    de.muenchen.ehrenamtjustiz.eai.personeninfo.processor.CountingProcessor: TRACE
+    de.muenchen.ehrenamtjustiz.eai.personeninfo.filter.RequestResponseLoggingFilter: DEBUG
+    org.apache.camel.Tracing: TRACE
+    org.springframework.boot.context.embedded.tomcat.TomcatEmbeddedServletContainer: INFO
+    de.muenchen.ehrenamtjustiz.eai.personeninfo: DEBUG
+    route.interceptedFrom: WARN
+    org.apache.cxf.services: INFO
+```
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)